### PR TITLE
Update output example

### LINKED ISSUE #34370: `dotnet sdk check` works well but... how to _update_ the out-of-date SDKs?
This is really a very minor issue, but it would help if you could add a link to the relevant documentation page where you learn how to _update_ the SDKs that `dotnet sdk check` reports as having available patches...

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0a159bb9-0caa-def6-76f4-a96bc7ac831f
* Version Independent ID: 2166d4b2-7ba1-d60b-fc7e-f69c3bc10b9f
* Content: [dotnet sdk check command - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-sdk-check)
* Content Source: [docs/core/tools/dotnet-sdk-check.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-sdk-check.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #34370 

[Internal review URL](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-sdk-check?branch=tdykstra-patch-1)

### CODE REVIEW

File: docs/core/tools/dotnet-sdk-check.md [hunk 1]
```diff
@@ -29,30 +29,24 @@ The `dotnet sdk check` command makes it easier to track when new versions of the
...
 ```output
-.NET SDKs:
-Version                         Status
------------------------------------------------------------
-2.1.816                         Up to date.
-2.2.401                         .NET 2.2 is out of support.
-3.1.410                         Up to date.
-5.0.204                         Up to date.
-5.0.301                         Up to date.
+3.1.426      .NET 3.1 is out of support.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-sdk-check.md:32]:
```suggestion
.NET SDKs:
3.1.426      .NET 3.1 is out of support.
```

File: docs/core/tools/dotnet-sdk-check.md [hunk 2]
```diff
@@ -30,29 +30,24 @@ Here's an example of output from the command:
 
 ```output
 .NET SDKs:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-sdk-check.md:32]:
```suggestion
Version      Status
----------------------------------------
.NET SDKs:
```

File: docs/core/tools/dotnet-sdk-check.md [hunk 3]
```diff
@@ -29,30 +29,27 @@ The `dotnet sdk check` command makes it easier to track when new versions of the
 Here's an example of output from the command:
 
 ```output
+Version      Status
+----------------------------------------
 .NET SDKs:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-sdk-check.md:34]:
```suggestion
.NET SDKs:
Version      Status
----------------------------------------
```

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Pull request merged**